charts/harmony-chart/values.yaml

```diff
@@ -364,3 +364,186 @@ velero:
+  podMonitor:
+    enabled: false
+  tolerations:
+  - operator: Exists
+  logLevel: "info"
+  customConfig:
+    data_dir: /vector-data-dir
+    api:
+      enabled: false
+      address: 0.0.0.0:8686
+      playground: false
+
+    sources:
+      kubernetes_tutor_logs:
+        type: kubernetes_logs
+        extra_namespace_label_selector: app.kubernetes.io/managed-by=tutor
+      kubernetes_global_logs:
+        type: kubernetes_logs
+        extra_namespace_label_selector: app.kubernetes.io/managed-by!=tutor
+        extra_field_selector: |
+          metadata.labels."app.kubernetes.io/name"=ingress-nginx,
+          metadata.labels."app.kubernetes.io/name"=cert-manager
+
+    transforms:
+
+      # Extract logs from Open edX applications
+      application_logs:
+        type: filter
+        inputs:
+        - kubernetes_tutor_logs
+        condition: '!contains(string!(.message), "[tracking]")'
+
+      # Filter out application and global logs whose message is empty to prevent Vector process crash when sending logs to Cloudwatch
+      # More details in https://github.com/vectordotdev/vector/issues/15539
+      typed_application_logs:
+        type: remap
+        inputs:
+        - application_logs
+        source: |-
+          .type = "application"
+        drop_on_error: true
+        drop_on_abort: true
+
+      typed_global_logs:
+        type: remap
+        inputs:
+        - kubernetes_global_logs
+        source: |-
+          .type = "global"
+        drop_on_error: true
+        drop_on_abort: true
+
+      # Appplication logs (OpenedX, ingress-nginx, cert-manager) can be send to cloudwatch
+      # or to s3. It will depend on user needs.
+      non_empty_logs:
+        type: remap
+        inputs:
+        - typed_application_logs
+        - typed_global_logs
+        source: |-
+          if is_empty(string!(.message)) {
+            log("Events with empty message are discarded", level: "info")
+            abort
+          }
+
+      # Extract tracking logs from Open edX applications
+      parsed_tracking_logs:
+        type: remap
+        inputs:
+        - kubernetes_tutor_logs
+        source: |-
+          parsed, err_regex = parse_regex(.message, r'^.* \[tracking\] [^{}]* (?P<tracking_message>\{.*\})$')
+          if err_regex != null {
+            abort
+          }
+          message = parsed.tracking_message
+          parsed_json, err_json = parse_json(parsed.tracking_message)
+          if err_json != null {
+            log("Unable to parse JSON from tracking log message: " + err_json, level: "info")
+            abort
+          }
+          time, err_timestamp = parse_timestamp(parsed_json.time, "%+")
+          if err_timestamp != null {
+            log("Unable to parse timestamp from tracking log message: " + err_timestamp, level: "info")
+            abort
+          }
+          .time = time
+          .message = message
+          .type = "tracking"
+
+      # Example ClickHouse Filter
```

Comment:
I would put this example (and the one in L533) into the https://github.com/openedx/openedx-k8s-harmony/blob/main/values-example.yaml. From helm chart point of view, this looks good to me, though I didn't check the setup.